Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/ScheduleAdapterItem.kt

```diff
@@ -1,18 +1,21 @@
 package com.mentalmachines.droidcon_boston.views.agenda
 
 import android.support.v4.content.ContextCompat
+import android.support.v7.widget.RecyclerView
 import android.util.Log
 import android.util.TypedValue
 import android.view.View
 import android.widget.ImageView
 import android.widget.TextView
 import com.bumptech.glide.Glide
+import com.bumptech.glide.load.engine.DiskCacheStrategy
```

Comment:
unused import?

Reply:
oops..forgot about that.  Was trying to see if setting the caching to All would let Glide display images on the agenda detail screen.  It supposedly downloads the full size image so it should just need to resize it but doesn't... :-P